### PR TITLE
core: fix api in a fuzzing test

### PR DESCRIPTION
**Pull request by ligurio**
API has been changed in d1ddf7a6c7781ae4963ebfc3070a4b7fd5d1c97d ("[crypto,test] update to new cert/crypto API").

### CODE REVIEW

File: libfreerdp/core/test/TestFuzzCryptoCertificateDataSetPEM.c [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
+	char *pem = calloc(Size + 1, sizeof(char));
+	if (!pem)
 		goto cleanup;
+	pem[Size] = '\0';
 
-	memcpy(buf, Data, Size);
-	if (!certificate_data_set_pem(cert_data, buf))
-	{
+	rdpCertificateData* data = NULL;
+	data = freerdp_certificate_data_new_from_pem("somehost", 1234, pem, strlen(pem));
```
Comment (akallabeth) [inline libfreerdp/core/test/TestFuzzCryptoCertificateDataSetPEM.c:13]: in this case better use `strnlen` as you know the maximum size of the string

**Comment by akallabeth** [comment]
@freerdp-bot test

File: libfreerdp/core/test/TestFuzzCryptoCertificateDataSetPEM.c (hunk 1 above)
Comment (ligurio) [reply libfreerdp/core/test/TestFuzzCryptoCertificateDataSetPEM.c]: right, fixed

**Comment by akallabeth** [comment]
@freerdp-bot test

**Comment by ligurio** [comment]
@akallabeth what was wrong with formatting?

```
+ echo 'Detected formatting issues in this pull request:'
Detected formatting issues in this pull request:
+ echo -----------------------------------------------
-----------------------------------------------
+ echo ''

+ cat /tmp/tmp.y7Kd35j3sc
changed files:
    libfreerdp/core/test/TestFuzzCryptoCertificateDataSetPEM.c
+ git diff --numstat
1	1	libfreerdp/core/test/TestFuzzCryptoCertificateDataSetPEM.c
+ git diff
diff --git a/libfreerdp/core/test/TestFuzzCryptoCertificateDataSetPEM.c b/libfreerdp/core/test/TestFuzzCryptoCertificateDataSetPEM.c
index 75c3e6133..bd67d6347 100644
--- a/libfreerdp/core/test/TestFuzzCryptoCertificateDataSetPEM.c
+++ b/libfreerdp/core/test/TestFuzzCryptoCertificateDataSetPEM.c
@@ -4,7 +4,7 @@
 
 int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
 {
-	char *pem = calloc(Size + 1, sizeof(char));
+	char* pem = calloc(Size + 1, sizeof(char));
 	if (pem == NULL)
 		goto cleanup;
 	memcpy(pem, Data, Size);
```

**Comment by akallabeth** [comment]
@ligurio you already posted the diff ;)
`char *pem` vs `char* pem` ;)

**Comment by ligurio** [comment]
(CodeQL / Analyze (cpp) failed due to infrastructure problems)

**Review by hardening — APPROVED** [review]

**Pull request merged**